packages/codec/lib/format/errors.ts

```diff
@@ -814,6 +814,7 @@ export type ReadError =
   | ReadErrorStack
   | ReadErrorBytes
   | ReadErrorStorage
+  | ReadErrorStorageDeliberate
```

Comment:
Can we call this something that indicates the read error is due to unhydrated trace? Maybe `ReadErrorStorageNotTraced`?

Reply:
Well, notionally it would occur whenever someone decided to pass in `null`, for whatever reason.  That's why my other suggestion was `NoStorageSuppliedError`.

Reply:
Ah cool yes I prefer that!

Reply:
OK I did that, although instead of `NoStorageSuppliedError` I called it `StorageNotSuppliedError`.

Note I left unchanged the name of the function that digs for these.